### ISSUE TITLE
We need to accommodate email verification workflows on mobile

### ISSUE THREAD

**Issue by edjiang** [stormpath/stormpath-framework-spec#116]
If I configure a Stormpath integration like so for mobile:

`produces: application/json`

The `/verify` endpoint will only listen for things preferring application/json, and won't respond to browser requests. 

However, the email sent will usually be in a mobile email client, which will open a browser (which prefers HTML). This request will fall through and 404.

Two solutions:
1. Suggest that the user uses the api.stormpath.com verify endpoint. Ensure that any authentication attempts refresh the cached account object, so it doesn't reject the authentication request as "UNVERIFIED".
2. Leave the `/verify` endpoint enabled for all response types? 

@robertjd, this is based on the slack conversation. Toughts?

**Comment by robertjd** [comment]
This is an issue.  (1) won't work, cuz that page is ugly and can't redirect the user back to the mobile application.  (2) may work, I need to check in again with our SPA strategies to make sure it's not going to break anything.

**Comment by edjiang** [comment]
I'm OK with 1 not redirecting the user back to mobile. If it's ugly, that's a quick fix.

**Comment by robertjd** [comment]
If you want to suggest (1) for mobile that's your call, but it feels lame.

**Comment by edjiang** [comment]
Yeah, redirecting back to mobile is actually quite a problem, since it's hard for a web browser to know whether or not a mobile app is installed. I think most apps don't deal with this and thus leave you on the website when you click the verification link on mobile. Some do a better job, but the techniques to accomplish this are varied.
